core/src/main/java/ar/com/wolox/wolmo/core/util/ToastUtils.java

```diff
@@ -23,12 +23,46 @@
 
 import android.widget.Toast;
 
+/**
+ * An utility class to work with Android's {@link Toast} messages
+ */
 public class ToastUtils {
-    public static void showToast(int resId) {
+
+    /**
+     * Displays a text message from a resource ID inside a {@link Toast}, briefly
+     *
+     * @param resId A resource ID from a {@link String} with the message to be displayed
+     */
+    public static void show(int resId) {
```

Comment:
Good idea, in fact, I added a `@StringRes` annotation which enforces the use of a String identifier (not just any identifier).